iidm/iidm-criteria/src/main/java/com/powsybl/iidm/criteria/PropertyCriterion.java

```diff
@@ -71,18 +70,11 @@ private boolean filterIdentifiable(Identifiable<?> identifiable) {
 
     private boolean filterEquipement(Identifiable<?> identifiable, IdentifiableType type) {
         switch (type) {
-            case STATIC_VAR_COMPENSATOR:
-            case SHUNT_COMPENSATOR:
-            case BUSBAR_SECTION:
-            case GENERATOR:
-            case DANGLING_LINE:
-            case LOAD:
-            case BATTERY:
+            case STATIC_VAR_COMPENSATOR, SHUNT_COMPENSATOR, BUSBAR_SECTION, GENERATOR, DANGLING_LINE, LOAD, BATTERY:
                 return filterSubstationOrVoltageLevel(((Injection) identifiable).getTerminal().getVoltageLevel());
             case SWITCH:
                 return filterSubstationOrVoltageLevel(((Switch) identifiable).getVoltageLevel());
-            case TWO_WINDINGS_TRANSFORMER:
-            case LINE:
+            case TWO_WINDINGS_TRANSFORMER, LINE:
```

Comment:
I dont know if it is better but with java 17 switch/case has a new form
```java
switch (args[0]) {
    case "A" -> System.out.println("Parameter is A");
    case b -> System.out.println("Parameter is b");
    default -> System.out.println("Parameter is unknown");
};
```

Reply:
Done